src/main/java/org/opensrp/repository/postgres/ClientsRepositoryImpl.java

```diff
@@ -383,7 +383,11 @@ private ClientMetadata createMetadata(Client client, Long clientId) {
 			clientMetadata.setFirstName(client.getFirstName());
 			clientMetadata.setMiddleName(client.getMiddleName());
 			clientMetadata.setLastName(client.getLastName());
-			
+
+			if(clientId != null){
```

Comment:
Do we need to add the complementary branch of this if condition for created at?

Reply:
Added it here https://github.com/OpenSRP/opensrp-server-core/blob/acbdf68075023cfb16a524f1ace88a6218c69b42/src/main/java/org/opensrp/repository/postgres/ClientsRepositoryImpl.java#L142